src/Compilers/CSharp/Test/Semantic/Semantics/LambdaTests.cs

```diff
@@ -6310,6 +6310,26 @@ class A : Attribute { }
                 );
         }
 
+        [Fact]
+        [WorkItem(60661, "https://github.com/dotnet/roslyn/issues/60661")]
+        public void KeywordParameterName()
+        {
+            var source =
+@"using System;
+class Program
+{
+    static void Main()
+    {
+        Action<int> a = int => { };
```

Comment:
Consider adding another test for `public`. Currently the behavior for `int` (stack exhausting) is different than `public` (too many syntax errors).

Also consider testing with `@int` (not too important)

Reply:
Added a parsing test for a full syntax tree with `public`.